src/Testcontainers.Gitlab/GitlabBuilder.cs

```diff
@@ -0,0 +1,78 @@
+ namespace Testcontainers.Gitlab;
+ 
+ /// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+ [PublicAPI]
+ public sealed class GitlabBuilder : ContainerBuilder<GitlabBuilder, GitlabContainer, GitlabConfiguration>
+ {
+     /// <summary>
+     /// This is the default image for gitlab community edition.
+     /// </summary>
+     public const string GitlabImage = "gitlab/gitlab-ce";
```

Comment:
Can we please pin the version?

Reply:
Sure thing. I tested with the latest image, with is version 16.11.1 .